<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1623</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_mango_loco">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mango Loco</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mango_loco">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_assault">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Assaulted :,(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_assaulted">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ultra_fiesta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ultra Fiesta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ultra_fiesta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bad_apple">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bad Apple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bad_apple">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>430</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="dynamicWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_warenkorb">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>40</y>
      <width>351</width>
      <height>711</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bestellen">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>800</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>bestellen</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1623</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
